crates/harp/src/error.rs

```diff
@@ -101,6 +101,7 @@ impl fmt::Display for Error {
                 ..
             } => {
                 if let Some(code) = code {
+                    let code = truncate_lines(code.clone(), 10);
```

Comment:
```suggestion
                    let code = truncate_lines(code.clone(), 50);
```

I would be ok with this too